src/pages/signin/ChangeExpensifyLoginLink.js

```diff
@@ -5,7 +5,7 @@ import PropTypes from 'prop-types';
 import Str from 'expensify-common/lib/str';
 import Text from '../../components/Text';
 import styles from '../../styles/styles';
-import {restartSignin} from '../../libs/actions/Session';
```

Comment:
I think this is the only place this method was used, so can you please remove the method completely?

Reply:
Sure.